src/event/handlers/CommandParserHandler.ts

```diff
@@ -1,7 +1,8 @@
 import { Constants, Message } from "discord.js";
-import { COMMAND_PREFIX } from "../../config.json";
 import EventHandler from "../../abstracts/EventHandler";
 import CommandFactory from "../../factories/CommandFactory";
```

Comment:
I didn't change all `config.json` calls to `getConfigValue` calls because I didn't know if it was the right thing to do. Should we only change it for variables that need to be mocked? Or should all calls be changed?